content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -20,12 +20,12 @@ do.
 ## Requirements for identity verification
 
 {: .number-list}
-1. Your [drivers license or state ID card.](/help/verify-your-identity/accepted-identification-documents/) You can upload a photo of your ID by phone or by computer.
-1. Social Security number
-1. Your phone number 
-    * If we can't verify your phone number, you can [verify by mail](/help/verify-your-identity/verify-your-address-by-mail/) instead. This takes 5 to 10 days.
-1. Taking a photo of yourself
-Some services need you to take a photo of yourself with a phone to verify your identity online. If you don't have a phone with a camera, you can [verify your identity in person.](/help/verify-your-identity/verify-your-identity-in-person/)
+1. Your [driver's license or state ID card.](/help/verify-your-identity/accepted-identification-documents/) You can upload a photo of your ID by phone or by computer.
+2. Social Security number
+3. Your phone number
+   In some cases, you can [verify by mail](/help/verify-your-identity/verify-your-address-by-mail/) instead.
```

Comment:
In addition to the copy change, I made this no longer a bullet point, to match 4 below. See [this Slack thread](https://gsa-tts.slack.com/archives/CNCGEHG1G/p1722544159925619?thread_ts=1722528808.044209&cid=CNCGEHG1G).